### ISSUE TITLE
Failing runtestRecoLocalCaloHGCalRecProducers (needs cleanup)

### ISSUE THREAD

**Issue by davidlt** [cms-sw/cmssw#14921]
`runtestRecoLocalCaloHGCalRecProducers` test is failing:
```
===== Test "runtestRecoLocalCaloHGCalRecProducers" ====
argument 0: runtestRecoLocalCaloHGCalRecProducers
argument 1: /bin/bash
argument 2: RecoLocalCalo/HGCalRecProducers/test
argument 3: runtests.sh
shell is: /bin/bash
Current directory is: /tmp/cmsbuild/workspace/ib-run-qa/CMSSW_8_1_X_2016-06-18-1100
topdir is: /tmp/cmsbuild/workspace/ib-run-qa/CMSSW_8_1_X_2016-06-18-1100
testdir is: /tmp/cmsbuild/workspace/ib-run-qa/CMSSW_8_1_X_2016-06-18-1100/src/RecoLocalCalo/HGCalRecProducers/test
tmpdir is: /tmp/cmsbuild/workspace/ib-run-qa/CMSSW_8_1_X_2016-06-18-1100/tmp/slc6_amd64_gcc493
testbin is: /tmp/cmsbuild/workspace/ib-run-qa/CMSSW_8_1_X_2016-06-18-1100/test/slc6_amd64_gcc493
Running script: /tmp/cmsbuild/workspace/ib-run-qa/CMSSW_8_1_X_2016-06-18-1100/src/RecoLocalCalo/HGCalRecProducers/test/runtests.sh
----- Begin Fatal Exception 18-Jun-2016 16:26:18 CEST-----------------------
An exception of category 'ConfigFileReadError' occurred while
   [0] Processing the python configuration file named /tmp/cmsbuild/workspace/ib-run-qa/CMSSW_8_1_X_2016-06-18-1100/src/RecoLocalCalo/HGCalRecProducers/test/testHGCalRecoLocal_cfg.py
Exception Message:
python encountered the error: <type 'exceptions.ImportError'>
No module named GeometryExtended2023LRecoReco_cff
----- End Fatal Exception -------------------------------------------------
Failure using testHGCalRecoLocal_cfg.py: status 90
status = 23040

---> test runtestRecoLocalCaloHGCalRecProducers had ERRORS
```

`GeometryExtended2023LRecoReco_cff.py` and `GeometryExtended2023LReco_cff.py` has been deleted a month ago.
```
commit 11e9b9824533c8dc4ffc8fd8bf04ee285a2d1603
Author: erica brondolin <erica.brondolin@cern.ch>
Date:   Tue May 17 17:58:13 2016 +0200

    avoiding file multiplication in geometry cfg

 .../python/GeometryExtended2023GRecoReco_cff.py    |   52 --------------------
 .../python/GeometryExtended2023GReco_cff.py        |   12 -----
 .../python/GeometryExtended2023LRecoReco_cff.py    |   52 --------------------
 .../python/GeometryExtended2023LReco_cff.py        |   12 -----
 .../StandardSequences/python/GeometryConf.py       |    4 +-
 5 files changed, 2 insertions(+), 130 deletions(-)
```

Cleanup list:
```
RecoLocalCalo/HGCalRecProducers/test/testHGCalRecoLocal_cfg.py:process.load('Configuration.Geometry.GeometryExtended2023LRecoReco_cff')
RecoLocalTracker/SiPhase2Clusterizer/test/ClustersValidationTest_flat_cfg.py:process.load('Configuration.Geometry.GeometryExtended2023LRecoReco_cff')
RecoLocalCalo/HGCalRecProducers/test/testHGCalRecoLocal_cfg.py:process.load('Configuration.Geometry.GeometryExtended2023LReco_cff')
```

**Comment by davidlt** [comment]
assign reconstruction

**Comment by lgray** [comment]
Fixed in: https://github.com/cms-sw/cmssw/pull/14872

**Comment by smuzaffar** [comment]
thanks @lgray

**Comment by davidlt** [comment]
But it doesn't clean `RecoLocalTracker/SiPhase2Clusterizer/test/ClustersValidationTest_flat_cfg` Thus we still have one more file referencing old stuff.

**Comment by thomaslenzi** [comment]
I opened PR #14931 to fix this issue.

**Comment by slava77** [comment]
+1

fixed in #14872 and then a related topic is fixed in #14931
